# Tarea3

Mediante la clase Exception creo una clase que hereda de ella,llamada CorreoException la cual tiene el metodo __init__ sobreescrito para poder pasarle un mensaje que se mostrará en la consola cuando salte la excepción.El archivo tare3.py uso la estructura try-except para manejar la excepcion CorreoException.Por último el archivo funciones.py contiene una funcion llamada verificar(email) que contiene un solo parametro el cual va a ser una cadena de caracteres ,en la cual se comrueba que tiene el formaro de un correo electrónico.
